lib/partner/partnerFields.js

```diff
@@ -28,6 +31,11 @@ const PartnerTypeFields = {
     type: new GraphQLNonNull(GraphQLString),
     description: 'URL for partner logo',
   },
+  jobs: {
+    type: new GraphQLList(JobType),
+    description: 'List of partner jobs',
```

Comment:
Yep, sorry should have made more clear in the description. This is in relation to [this](https://trello.com/c/EaiIO5ck/28-sponsors-page-react-london-conf) ticket on trello